class PoeConnectionError(Exception):
    pass


class PoeBotError(Exception):
    pass
